pilvytis/order_status_tracker.go

```diff
@@ -124,9 +140,21 @@ func (t *StatusTracker) update() error {
 		}
 		for _, newOrder := range newOrders {
 			order := t.getOrCreate(newOrder.ID, newOrder.Identity)
+			changed := false
 			if order.Status != newOrder.Status {
 				order.Status = newOrder.Status
-				t.eventBus.Publish(AppTopicOrderStatusChanged, AppEventOrderStatusChanged{*order})
+				changed = true
+			}
```

Comment:
nitpick: Maybe move these 3 changed checks in to a single func which would then also be tested if you wanted to? Code would also read nicer imo.

Reply:
Done